app/assets/stylesheets/style.css

```diff
@@ -263,3 +267,71 @@ li {
   transform: scale(1.05);
   transition: all 1s ease;
 }
+
+.shared_button {
+  background-color: #222;
+  padding: 10px;
+  border-radius: 5px;
+  display: flex;
+  justify-self: center;
+  color: white;
+  align-self: center;
+  width: 500px;
+  justify-content: center;
+}
+
+.edit_user {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
+
+.new_user {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
```

Comment:
- [optional ] Please kindly avoid code repetition in order to make your codebase cleaner. 👍 **Note: The same is applicable to line 297 to 325 as well** 🙏